src/common/datatypes/Value.cpp

```diff
@@ -678,6 +678,11 @@ const DataSet& Value::getDataSet() const {
   return *(value_.gVal);
 }
 
+DataSet& Value::getMutableDataSet() {
+  CHECK_EQ(type_, Type::DATASET);
```

Comment:
DCHECK

Reply:
All checks in this Value.cpp are `CHECK_EQ`. I discussed with @jievince, it seems `DCHECK_EQ` is enough here. Let's leave it for another PR.

Reply:
https://github.com/vesoft-inc/nebula/issues/4971

Reply:
ACK.